spec/support/database_cleaner.rb

```diff
@@ -7,10 +7,13 @@
   # Clean the database before running tests. Setup as per
   # https://github.com/DatabaseCleaner/database_cleaner#rspec-example
   config.before(:suite) do
-    DatabaseCleaner.strategy = :transaction
     DatabaseCleaner.clean_with(:truncation)
   end
 
+  config.before(:each) do
+    DatabaseCleaner.strategy = :truncation
```

Comment:
Is this fixing the issue with the unique keys? The commit message sounded like we weren't sure if it had solved the problem or not.

Reply:
It does solve it yes :)